active_model_serializers.gemspec

```diff
@@ -47,4 +47,9 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency 'bundler', '~> 1.6'
   spec.add_development_dependency 'timecop', '>= 0.7'
+
+  # development dependencies, cause... how can you not use these? :-)
+  spec.add_development_dependency "pry-byebug"
+  spec.add_development_dependency "awesome_print"
```

Comment:
So, two very opinionated comments

1. I try to only include dev deps in the gemspec if they are required to run tests.  It sort of doesn't matter because gem test is dead, but still
2. https://github.com/rails-api/active_model_serializers/pull/1017#discussion_r37602184 would let you use your own local gems